scripts/obtain/dotnet-install.sh

```diff
@@ -255,7 +255,7 @@ check_pre_reqs() {
         local librarypath=${LD_LIBRARY_PATH:-}
         LDCONFIG_COMMAND="$LDCONFIG_COMMAND -NXv ${librarypath//:/ }"
 
-        [ -z "$($LDCONFIG_COMMAND 2>/dev/null | grep libunwind)" ] && say_warning "Unable to locate libunwind. Probable prerequisite missing; install libunwind."
+        [ -z "$($LDCONFIG_COMMAND 2>/dev/null | grep zlib)" ] && say_warning "Unable to locate zlib. Probable prerequisite missing; please install zlib."
```

Comment:
were we missing zlib before? the warning that is.

Reply:
It appears that we were missing 'zlib' before...
See: https://docs.microsoft.com/en-us/dotnet/core/linux-prerequisites?tabs=netcore30#supported-linux-versions